File: verify/flow/break.py
def expect(*args):
    res = ""
    for i in range(10):
        if i == 5: break
        res += "{} ".format(i)
    res += '\n'
    for i in range(10):
        if i == 7: break
        for j in range(10):
            if i == j: break;
            res += "({},{}) ".format(i, j)
        res += '\n'

    i = 0;
    while True:
        if i == 10:
            res += "exit.\n"
            break;
        res += "{} ".format(i)
        i += 1
    return (0, res, "")

data = [
    [],
]

File: tests/flow/break.c
int main() {
    for (int i = 0; i < 10; i++) {
        if (i == 5) break;
        printf("%d ", i);
    }
    putchar('\n');
    for (int i = 0; i < 10; i++) {
        if (i == 7) break;
        for (int j = 0; j < 10; j++) {
            if (i == j) break;
            printf("(%d,%d) ", i, j);
        }
        putchar('\n');
    }
    int i = 0;
    while (1) {
        if (i == 10) {
            printf("exit.\n");
            break;
        }
        printf("%d ", i++);
    }
}
